hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -24,7 +24,7 @@ filesets:
       - lowrisc:ip:lc_ctrl_pkg
       - lowrisc:ip:edn_pkg
       - lowrisc:ip_interfaces:alert_handler_reg
-      - lowrisc:ip_interfaces:rstmgr_pkg
+      - lowrisc:opentitan:top_earlgrey_rstmgr_pkg
```

Comment:
This should be reverted since otherwise the ipgen flow does not work correctly.

Reply:
@vogelpi @andreaskurth

Reply:
if this did not work for you, please update your localfusesoc installation using 
```
pip install -r python_requirements.txt
```

Reply:
https://github.com/lowRISC/opentitan/pull/22082